### PR TITLE
Billing addon

### PR DESCRIPTION
**Pull request by piotrmilcarz**

### CODE REVIEW

File: lib/travis/hub/config.rb [hunk 1]
```diff
@@ -38,8 +38,9 @@ def jwt_key(type)
              repository:     { ssl_key: { size: 4096 } },
              queue:          'builds',
              limit:          { resets: { max: 50, after: 6 * 60 * 60 } },
-             notifications:  [],
-             auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth }
+             notifications:  [ 'billing' ],
+             auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth },
+             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth: ENV['BILLING_AUTH'] || 't0Ps3Cr3t' }
```
Comment (pavel-d) [inline lib/travis/hub/config.rb:43]: @piotrmilcarz, isn't the auth_key different in default dev env?

Comment (piotrmilcarz) [reply lib/travis/hub/config.rb]: Nice catch @pavel-d !
Thank you. Fixed now.

File: lib/travis/hub/config.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def jwt_key(type)
              limit:          { resets: { max: 50, after: 6 * 60 * 60 } },
              notifications:  [ 'billing' ],
              auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth },
-             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth: ENV['BILLING_AUTH'] || 't0Ps3Cr3t' }
+             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth_key: ENV['BILLING_AUTH_KEY'] || 't0Ps3Cr3t' }
```
Comment (shairyar) [inline lib/travis/hub/config.rb:43]: @piotrmilcarz thank you, just curious to know why hardcode the `auth` value to `t0Ps3Cr3t`

Comment (artursmirnov) [reply lib/travis/hub/config.rb:43]: @shairyar I'm guessing, it's just a mock for tests, read it as "topsecret" :)

Comment (piotrmilcarz) [reply lib/travis/hub/config.rb:43]: It was used just for testing purposes. I hope it's not one of used passwords in real world ;-)

**Pull request merged**